Ignition Plugin Error

Hi there! Thanks for the amazing work. I'm trying to put up a simple test using your software because I need to implement some flocking algorithms for a university project. The error I'm getting is the following
```
CMake Error at CMakeLists.txt:8 (find_package):
  By not providing "Findignition-plugin1.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "ignition-plugin1", but CMake did not find one.

  Could not find a package configuration file provided by "ignition-plugin1"
  with any of the following names:

    ignition-plugin1Config.cmake
    ignition-plugin1-config.cmake

  Add the installation prefix of "ignition-plugin1" to CMAKE_PREFIX_PATH or
  set "ignition-plugin1_DIR" to a directory containing one of the above
  files.  If "ignition-plugin1" provides a separate development package or
  SDK, be sure it has been installed.
```

I'm on pop-os 22.04 (everything that works on Ubuntu 22.04 has always worked until now). I've followed [this guide](https://www.bitcraze.io/2024/09/crazyflies-adventures-with-ros-2-and-gazebo/), installing gazebo harmonic and the bridge for ros humble. I've also tried, without success, to install the packages that are apparently missing. 

Thank you in advance

--- a/ros_gz_crazyflie_gazebo/CMakeLists.txt
+++ b/ros_gz_crazyflie_gazebo/CMakeLists.txt
@@ -4,14 +4,6 @@ project(ros_gz_crazyflie_gazebo)
 
 find_package(ament_cmake REQUIRED)
 
-find_package(ignition-cmake2 REQUIRED)
-find_package(ignition-plugin1 REQUIRED COMPONENTS register)
-set(GZ_PLUGIN_VER ${ignition-plugin1_VERSION_MAJOR})
-find_package(ignition-common4 REQUIRED COMPONENTS profiler)
-set(GZ_COMMON_VER ${ignition-common4_VERSION_MAJOR})
-find_package(ignition-gazebo6 REQUIRED)
-set(GZ_SIM_VER ${ignition-gazebo6_VERSION_MAJOR})
-
 
 install(
   DIRECTORY worlds/

